flutter/lib/src/native/cocoa/sentry_native_cocoa.dart

```diff
@@ -14,7 +14,7 @@ class SentryNativeCocoa extends SentryNativeChannel {
 
   @override
   int? startProfiler(SentryId traceId) {
-    final cSentryId = cocoa.SentryId.alloc(_lib)
+    final cSentryId = cocoa.SentryId1.alloc(_lib)
```

Comment:
this is wanted, right?

Reply:
yep, this is a change from https://github.com/getsentry/sentry-dart/pull/1986 which is not on the v7 branch anymore